### ISSUE TITLE
Any action should bump post

### ISSUE THREAD

**Issue by epicmonkey** [pepyatka/pepyatka-server#179]
Create/delete comment, like/unlike events should bump post.

**Comment by indeyets** [comment]
not as simple as that. there are very specific bump rules

**Comment by epicmonkey** [comment]
Indeed. We shall start with a simple approach and prepare method Post#bump that will eventually be updated to another solution.

**Comment by smallq-git** [comment]
Spec for feed behavior: https://docs.google.com/document/d/1oQKrLTH7GHi5lHACuTW6boKukiu8LQzVsOet6Ev4AM4/edit
(Feed, p.1.2)

**Comment by epicmonkey** [comment]
For the time being like does not bump post anymore: https://github.com/pepyatka/pepyatka-server/commit/e4f5271e62edfe87e67de757f93574277224e08a
